src/odemis/driver/kryoz.py

```diff
@@ -0,0 +1,159 @@
+# -*- coding: utf-8 -*-
+'''
+Created on 5 November 2020
+
+@author: Anders Muskens
+
+Copyright © 2020 Anders Muskens, Delmic
+
+This file is part of Odemis.
+
+Odemis is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License version 2 as published by the Free Software Foundation.
+
+Odemis is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with Odemis. If not, see http://www.gnu.org/licenses/.
+'''
+
+from __future__ import division
+
+import logging
+
+from odemis import model
+from odemis.util import to_str_escape
+
+import threading
+import socket
+
+DEFAULT_PORT = 5041
+
+
+class CoolerBasic():
+    '''
+    Basic cooler driver
+    '''
+
+    def __init__(self, name, role, host, port=DEFAULT_PORT, daemon=None, **kwargs):
+        """
+        host: (string) the TCP/IP hostname of the server
+        port: (int) the TCP/IP port of the server.
+
+        Raises:
+            ValueError if no scanner child is present
+        """
+        #super(Cooler, self).__init__(name, role, daemon=daemon, **kwargs)
```

Comment:
Why is this line commented out/still here? This class has no parent so inheriting would be hard anyway isn't it?

Reply:
Actually, please put this line. That's needed to have the component properly registered, and shared over Pyro.
It should look like:
`super(Cooler, self).__init__(name, role, **kwargs)`